packages/site/src/content/InputNumber/InputNumber.props-mobile.json

```diff
@@ -669,10 +669,10 @@
           "name": "() => string"
         }
       },
-      "__@iterator@24029": {
```

Comment:
I am not 100% why this change and the one for thumbnail list exists.
Maybe @jdeichert knows 🤔